### PR TITLE
Quickbuild (#40 & Base64CAs (#38) pull requests combined

### PR DESCRIPTION
**Pull request by MXClyde**
Combination of pull request #40 and #38 .
Merge conflicts have been removed after combining these pull requests.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,24 +3,17 @@
...
-RUN apt-get -q -y update && \
-  DEBIAN_FRONTEND=noninteractive apt-get upgrade -q -y && \
-  DEBIAN_FRONTEND=noninteractive apt-get install -q -y python wget curl libgdiplus libpq5
-# RUN apk update && \
-#     apk add --no-cache python2 curl openjdk8 postgresql-client && \
-#     apk add libgdiplus --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/
-
 # Build-time variables
 ARG BUILD_PATH=project
+ARG DD_API_KEY
```
Comment (dudk) [inline Dockerfile]: Remove for now, please

File: Dockerfile [hunk 2]
```diff
@@ -3,24 +3,17 @@
...
 
 # Checkout CF Build-pack here
 RUN mkdir -p buildpack/.local && \
-   (wget -qO- https://github.com/mendix/cf-mendix-buildpack/archive/v1.9.1.tar.gz \
+   (wget -qO- https://github.com/mendix/cf-mendix-buildpack/archive/master.tar.gz \
```
Comment (dudk) [inline Dockerfile]: We should rely on released versions, so please use latest release
https://github.com/mendix/cf-mendix-buildpack/archive/v1.9.2.tar.gz

Comment (MXClyde) [reply Dockerfile]: Changed.

**Comment by MXClyde** [comment]
Will close this request to make changes.

**Pull request closed without merging**